pom.xml

```diff
@@ -49,7 +49,9 @@
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
 
-        <asciidoctorj.version>1.5.3.2</asciidoctorj.version>
+        <asciidoctorj.version>1.6.0-alpha.3</asciidoctorj.version>
```

Comment:
@marko-bekhta looks like it might even be unnecessary to do the work ourselves: https://github.com/asciidoctor/asciidoctor-ant/issues/56#issuecomment-298697737

Let's see how it goes on the asciidoctor-ant front.